green_city/tests/api/get_eco_news_dislikes_test.py

```
@@ -0,0 +1,92 @@
+
+from green_city.src.config import API_BASE_URL_8085, ENDPOINTS
+
+
+def test_get_eco_news_dislikes_count_success(create_news, auth_token):
+    eco_news_id = create_news
+    headers = {"Authorization": f"Bearer {auth_token}"}
+
+    like_response = requests.post(
+        f"{API_BASE_URL_8085}{ENDPOINTS['create_eco_news']}/{eco_news_id}/like",
+        headers=headers
+    )
```

Pylint (3.3.7)

[warning] Missing timeout argument for method 'requests.post' can cause your program to hang indefinitely

(W3101)